lib/lts/index.js

```diff
@@ -101,13 +101,43 @@ function ltsVersions(_time, currentDate) {
     });
 }
 
+/**
+ *
+ * @param {list of LTS version} ltsList
+ * Returns current LTS version from the list of all LTS available
```

Comment:
```suggestion
 * Returns current LTS version from the sorted list (old to new) of all available LTS versions.
```

Reply:
This summary is wrong. This function returns the latest LTS from the list of all LTS available.